runtime/proto/corfu_options.proto

```diff
@@ -17,6 +17,8 @@ message SchemaOptions {
     optional bool is_federated = 4;
     // Tag tables with unique stream listener tags for selectivity in receiving change notifications.
     repeated string stream_tag = 5;
+    // String of comma-separated nested secondary keys
+    optional string nested_secondary_keys = 6;
```

Comment:
I don't think so. I was not part of the design, I was already told of how it would look like, but one reason I could think of is simplification, as copying "[(org.corfudb.runtime.schema).nested_secondary_keys = " multiple times could be annoying. But I don't think there is any other special reason.